### PR TITLE
Wrap the Sierra description in <p> tags

### PR DESCRIPTION
**Pull request by alexwlchan**
Closes https://github.com/wellcomecollection/platform/issues/4900

I did include the example from the ticket as a test case while writing this patch, but it was a duplicate of an existing test, so I removed it before opening this PR.

### CODE REVIEW

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraDescription.scala [hunk 1]
```diff
@@ -12,50 +12,37 @@ object SierraDescription extends SierraDataTransformer with SierraQueryOps {
...
-  // The value comes from comes subfield $a concatenated with subfield $b.
+  //  - Join 520 ǂa and ǂb with a space
+  //  - Wrap resulting string in <p> tags
+  //  - Join each occurrence of 520 into description
   //
   // Notes:
-  //  - A bib may have multiple 520 records, in which case we join with spaces
-  //  - If $b is empty, we just use $a
-  //  - We never expect to see a record with $b but not $a
+  //  - Both ǂa (summary) and ǂb (expansion of summary note) are
```
Comment (warrd) [inline pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraDescription.scala:22]: What are these symbols? :thinking:  https://en.wikipedia.org/wiki/Palatal_click

**Review by warrd — APPROVED** [review]

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraDescription.scala (hunk 1 above)
Comment (alexwlchan) [reply pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraDescription.scala:22]: I’ve seen them used in MARC reference docs for subfield tags. I’m not sure there’s a consistent standard. 🤷‍♀️
